invenio_app_ils/documents/loaders/jsonschemas/document.py

```diff
@@ -75,6 +75,7 @@ class Meta:
     language = fields.Str()
     source = fields.Str()
     value = fields.Str(required=True)
+    type = fields.Str()
```

Comment:
what it is used for (maybe I missed something)?

Reply:
The document loader was missing the `type` field so the alternative title type info was discarded on create/update.